rules/windows/execution_posh_hacktool_authors.toml

```diff
@@ -0,0 +1,90 @@
+[metadata]
+creation_date = "2023/01/17"
+integration = ["windows"]
+maturity = "production"
+updated_date = "2023/01/17"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+
+[rule]
+author = ["Elastic"]
+description = """
+Detects known PowerShell offensive tooling author's name in PowerShell scripts. Attackers commonly use out-of-the-box
+offensive tools without modifying the code. This rule aim is to take advantage of that.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-windows.*", "logs-system.*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "Potential PowerShell HackTool Script by Author"
+note = """## Setup
+
+The 'PowerShell Script Block Logging' logging policy must be enabled.
+Steps to implement the logging policy with with Advanced Audit Configuration:
+
+```
+Computer Configuration >
+Administrative Templates >
+Windows PowerShell >
+Turn on PowerShell Script Block Logging (Enable)
+```
+
+Steps to implement the logging policy via registry:
+
+```
+reg add "hklm\\SOFTWARE\\Policies\\Microsoft\\Windows\\PowerShell\\ScriptBlockLogging" /v EnableScriptBlockLogging /t REG_DWORD /d 1
+```
+"""
+references = [
+    "https://github.com/atc-project/atc-data/blob/master/docs/Logging_Policies/LP_0109_windows_powershell_script_block_log.md",
+]
+risk_score = 47
+rule_id = "2553a9af-52a4-4a05-bb03-85b2a479a0a0"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Execution", "PowerShell"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.category:process and
+  powershell.file.script_block_text : (
+    "JosephBialek" or "mattifestation" or
```

Comment:
Do we need to add wildcards to these filters by chance or will these substrings be identified in the full keyword strings?